src/server.test.js

```diff
@@ -0,0 +1,34 @@
+const server = require('./server');
+const request = require('supertest');
+
+describe("example test", () => {
+
+    test("it should work", () => {
+        expect(1).toBe(1);
```

Comment:
Since you have added a valid test below, perhaps this can be removed since it doesn't actually test anything.

Reply:
Will do!